assemble/bin/accumulo-service

```diff
@@ -25,7 +25,8 @@ Usage: accumulo-service <service> <command>
 Services:
   gc          Accumulo garbage collector
   monitor     Accumulo monitor
-  master      Accumulo master
+  manager     Accumulo manager
+  master      Accumulo master (Deprecated)
```

Comment:
This isn't related to your change, but: should this list be organized alphabetically? It's roughly, but not quite, alphabetical.